tikv-client/pom.xml

```diff
@@ -120,6 +120,12 @@
             <version>${grpc.version}</version>
             <scope>test</scope>
         </dependency>
+      <dependency>
+        <groupId>org.apache.commons</groupId>
+        <artifactId>commons-lang3</artifactId>
```

Comment:
only `StringUtils.removeStart` and `StringUtils.removeEnd` is used

do we really need this package ?